incubator/spring-cloud-data-flow/templates/_helpers.tpl

```diff
@@ -0,0 +1,34 @@
+{{/* vim: set filetype=mustache: */}}
```

Comment:
Our best practices suggest to use `fullname` for resource names and `name` for `app` labels. You use `shortname`, which is an adapted version of `fullname`, for both. I think it is fine to abbreviate the name as you do but still stick to our best practices. I'd suggest you create two templates `scdf.fullname` and `scdf.name`.